edx_reverification_block/xblock/static/js/checkpoint_edit.js

```diff
@@ -44,14 +44,14 @@ function CheckpointEditBlock(runtime, element) {
                     runtime.notify('save', {state: 'end'});
                 } else {
                     runtime.notify('error', {
-                        title: gettext('Re-Verification Save Error'),
+                        title: gettext('Reverification Save Error'),
                         message: gettext('An unexpected error occurred while saving.')
```

Comment:
Didn't see this message earlier either. Let's go with this:

An unexpected error occurred. Select Save to try again.